src/test/kotlin/org/rust/ide/navigation/goto/RustGotoSuperHandlerTest.kt

```diff
@@ -0,0 +1,20 @@
+package org.rust.ide.navigation.goto
+
+import com.intellij.lang.CodeInsightActions
+import org.rust.lang.RustLanguage
+import org.rust.lang.RustTestCaseBase
+
+class RustGotoSuperHandlerTest : RustTestCaseBase() {
+    override val dataPath = "org/rust/ide/navigation/fixtures/goto_super"
+
+    fun testGotoSuper() {
+        myFixture.copyFileToProject("main.rs", "/main.rs")
```

Comment:
yes i remove the `main.rs` file. the most tricky part of this plugin project is cargo stuffs i think :sweat_smile:, maybe i can understand the whole picture later. it make sense now, thx :)